src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -241,7 +242,7 @@ function MoneyRequestView({
     const getPendingFieldAction = (fieldPath: TransactionPendingFieldsKey) => transaction?.pendingFields?.[fieldPath] ?? pendingAction;
 
     const getErrorForField = useCallback(
-        (field: ViolationField, data?: OnyxTypes.TransactionViolation['data']) => {
+        (field: ViolationField, data?: OnyxTypes.TransactionViolation['data'], policyHasDependentTags?: boolean) => {
```

Comment:
Can we make the default value of `policyHasDependentTags` be false here and in `getViolationsForField`, and in `getViolationsForField` use `policyHasDependentTags` instead of `!!policyHasDependentTags`?

Reply:
@cead22 Done 😄